src/server.cpp

```diff
@@ -571,8 +571,8 @@ void Server::stop()
 void Server::step(float dtime)
 {
 	// Limit a bit
-	if (dtime > 2.0)
-		dtime = 2.0;
+	if (dtime > 2.5)
+		dtime = 2.5;
```

Comment:
Should all these 2.5's be extracted to a named constant so that it could be tweaked easier?

Reply:
I don't think we want to tweak this often. But point definitely taken.
In fact we might want to have three different constants:
- server adjustment
- client adjustment
- collision adjustment

Reply:
Actually. The collision delay must the same as server delay on the server, and the same as the client delay on the client. Which means one constant is better, otherwise it is confusing.
The interesting part is: Where should I put that constant? Could be src/constants.h but we only put very general things in there.